src/pages/todo-app.jsx

```diff
@@ -62,8 +108,35 @@ const TodoApp = () => {
                                 <rect x="14" y="6" width="13" height="2" transform="rotate(90 14 6)" fill="white" />
                             </svg>
                         </button>
+                   
                     </div>
+                         
+                   { tasks.map((task)=> {
+                       const { id, description } = task;
+                       return(
+                           <div className="task" key={id}>
+                               <p>{description}</p>
+                               <div>
+                                   <button> <svg
+                    className="delete"
+                    stroke="currentColor"
+                    fill="#eb6d7d"
+                    strokeWidth="0"
+                    viewBox="0 0 448 512"
+                    height="1em"
+                    width="1em"
+                    xmlns="http://www.w3.org/2000/svg"
+                  >
+                    <path d="M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z"></path>
+                  </svg></button>
+                
+                               </div>
+                           </div>
+                       )
```

Comment:
create a component that takes `description` as a prop and abstract this selected code out into that component and call it `TaskItem`. That way this can be rewritten as 
```
tasks.map(({ id, description })=> <TaskItem key={id} description={description} />)
```
it would make your code look cleaner